ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/privacy/eea/EeaGetTransactionReceipt.java

```diff
@@ -102,8 +102,7 @@ public JsonRpcResponse response(final JsonRpcRequest request) {
       privacyGroupId = receiveResponse.getPrivacyGroupId();
     } catch (Exception e) {
       LOG.error("Failed to fetch transaction from Enclave with error " + e.getMessage());
-      return new JsonRpcSuccessResponse(
-          request.getId(), JsonRpcError.PRIVATE_TRANSACTION_RECEIPT_ERROR);
+      return new JsonRpcSuccessResponse(request.getId(), null);
```

Comment:
Should the log error be removed? Are there any exceptions which actually are errors?

Reply:
There are no errors - an exception here means you do not have access to the private transaction you are trying to get the receipt for.